helm_image_updater/cli.py

```diff
@@ -176,7 +181,9 @@ def main():
             for tag in config.extra_tags
         )
 
-        if image_tag.startswith("dev-") or extra_tags_contains_dev:
+        if override_stack:
+            changes, missing_tags = update_stack_by_id(config, override_stack)
+        elif image_tag.startswith("dev-") or extra_tags_contains_dev:
```

Comment:
ak je zadany override_stack tak to stacky nevyhodnocuje automaticky ale update priamo na dany stack volanim update_stack_by_id